src/diffusers/pipelines/stable_cascade/pipeline_stable_cascade_combined.py

```diff
@@ -29,11 +29,10 @@
 TEXT2IMAGE_EXAMPLE_DOC_STRING = """
     Examples:
         ```py
-        >>> from diffusions import StableCascadeCombinedPipeline
-
-        >>> pipe = StableCascadeCombinedPipeline.from_pretrained("stabilityai/stable-cascade-combined", torch_dtype=torch.bfloat16).to(
-        ...     "cuda"
-        ... )
+        >>> import torch
+        >>> from diffusers import StableCascadeCombinedPipeline
+        >>> pipe = StableCascadeCombinedPipeline.from_pretrained("stabilityai/stable-cascade", variant="bf16", torch_dtype=torch.bfloat16)
```

Comment:
we should probably also make stable cascade combined pipeline work with auto pipeline (we just need to add "stable-cascade" into all the mappings
https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/auto_pipeline.py

then it can be used like this, we can use this in our main doc (for this example for combine pipeline here I think it is better to leave it as it is - is better to use `StableCascadeCombinedPipeline` explicitly here) 
```python
pipeline =  AutoPipelineForText2Image.from_pretrained(
    "stabilityai/stable-cascade", variant="bf16", torch_dtype=torch.bfloat16)
```